components/public-api/gitpod/experimental/v2/configuration.proto

```diff
@@ -0,0 +1,88 @@
+syntax = "proto3";
+
+package gitpod.experimental.v2;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v2";
+
+import "google/protobuf/timestamp.proto";
+import "gitpod/experimental/v2/pagination.proto";
```

Comment:
I cannot find such proto file

Reply:
it's in the parent commit. This PR is based on the other PR that includes it